backend/src/main/java/no/ntnu/database/controllers/TwoFactorRequests.java

```diff
@@ -0,0 +1,71 @@
+package no.ntnu.database.controllers;
+
+import java.util.Optional;
+import no.ntnu.database.entities.User;
+import no.ntnu.database.repositories.UserRepository;
+import no.ntnu.security.SecurityUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+/**
+ * Service for handling two-factor authentication operations.
+ */
+@Service("twoFactorRequests")
+public class TwoFactorRequests {
```

Comment:
This should be a service class inside the `services` package

Reply:
This was moved temporarily from service package for testing purpose. Has been moved back.